tests/TestCase.php

```diff
@@ -25,4 +25,14 @@
 abstract class TestCase extends BaseTestCase
 {
     use ExpectDeprecationTrait;
+
+    final public function testNotDefiningConstructor(): void
+    {
+        $reflection = new \ReflectionObject($this);
+
+        self::assertNotSame(
```

Comment:
whenever doing things like this, use the message parameter. JohnDoe will see that they done sth we didn't want, but without understanding "why" we don't want it.